.github/workflows/server-test-template.yml

```diff
@@ -56,6 +56,8 @@ jobs:
           docker compose --ansi never exec -T minio sh -c 'mkdir -p /data/mattermost-test';
           docker compose --ansi never ps
       - name: Run Tests
+        env:
+          BUILD_IMAGE: mattermost/mattermost-build-server:${{ steps.go.outputs.GO_VERSION }}
```

Comment:
Why is this moved down to only this step?

Reply:
This is the only step it is used so we do not need it globally declared in the job. 
Also I want to utilize the output variable and I cannot in the job env context  since the output is generated throughout the job